x/observer/client/cli/tx_update_observer.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"strconv"
+
+	errorsmod "cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/zetacore/x/observer/types"
+)
+
+func CmdUpdateObserver() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-observer [old-observer-address] [new-observer-address] [update-reason]",
+		Short: "Broadcast message add-observer",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			updateReasonInt, err := strconv.ParseInt(args[2], 10, 32)
+			if err != nil {
+				return err
+			}
+			updateReason, err := ParseUpdateReason(int32(updateReasonInt))
```

Comment:
Why not maybe use a string for the reason to convert?
- `tombstoned`
- `admin-update`
and returns an error if something else

Reply:
We have been using numbers for other status flags in the cli , just wanted to keep it uniform